graylog2-web-interface/src/components/welcome/RecentActivityList.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+
+import React, { useCallback, useMemo, useState } from 'react';
+import styled, { css } from 'styled-components';
+
+import { Table } from 'components/bootstrap';
+import { DEFAULT_PAGINATION, entityTypeMap } from 'components/welcome/Constants';
+import { NoSearchResult, PaginatedList, Spinner } from 'components/common';
+import { relativeDifference } from 'util/DateTime';
+import Routes from 'routing/Routes';
+import { Link } from 'components/common/router';
+import { StyledLabel } from 'components/welcome/EntityListItem';
+import { useRecentActivity } from 'components/welcome/hooks';
+import type { EntityItemType, RecentActivityType } from 'components/welcome/types';
+
+const ActionItemLink = styled(Link)(({ theme }) => css`
+  color: ${theme.colors.variant.primary};
+  &:hover {
+    color: ${theme.colors.variant.darker.primary};
+  }
+`);
```

Comment:
We had the topic of differently coloured links in the past and usually got feedback that it would look better / be easier to understand if we color links the common way. I do not think the current version looks bad. Although the default color would probably look better with the dark mode version:

![image](https://user-images.githubusercontent.com/46300478/213716461-6939f50f-7984-41cc-839f-48abd79fae4b.png)